docs/guides/upgrading.rst

```diff
@@ -454,6 +454,11 @@ Also several workarounds for very old views are no longer performed. Make these
  - Set ``$vars['name']`` instead of ``$vars['internalname']``.
  - Set ``$vars['id']`` instead of ``$vars['internalid']``.
 
+Removed libraries
+-----------------
+
+ - ``elgg.markdown``: replace with ``use \Michelf\MarkdownExtra``
```

Comment:
@Elgg/owners is our intention to keep providing MarkdownExtra as part of our API?